app/src/main/java/com/codingblocks/cbonlineapp/auth/onboarding/OnBoardingActivity.kt

```diff
@@ -35,30 +31,58 @@ class OnBoardingActivity : BaseCBActivity() {
         }
     }
 
-    private fun setAdapter() {
-        viewPager.apply {
-            adapter = mAdapter
+    private fun setupIntroSlider() {
```

Comment:
this function seems unnecessary to me instead setup the viewpager here also add Kdoc here as well.